src/Hedgehog/Extras/Test/Golden.hs

```diff
@@ -85,14 +94,15 @@ diffVsGoldenFile actualContent referenceFile = GHC.withFrozenCallStack $ do
         _        -> do
           H.note_ $ "Golden test failed against golden file: " <> referenceFile
```

Comment:
I think we should add info about both flags in the failure message. That would make diagnosis of the failure quicker. Right now I feel that new users are a bit confused by the golden failure.

Reply:
Added!